<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlayerControl</class>
 <widget class="QWidget" name="VideoPlayerControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>82</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="currentLabel">
       <property name="text">
        <string>2020-03-20T14:25:36.456Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="remainingLable">
       <property name="text">
        <string>-00:00:00.000</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="stopButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="media-playback-stop"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="skipBackwardButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="media-skip-backward"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="seekBackwardButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="media-seek-backward"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="playButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="media-playback-start"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="seekForwardButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="media-seek-forward"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="skipForwardButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="media-skip-forward"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Playing Speed:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
